bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Election.cs

```diff
@@ -112,29 +113,74 @@ public unsafe ElementModQ CryptoExtendedBaseHash
             }
         }
 
+        /// <summary>
+        /// Get a linked list containing the extended data of the election.
+        /// </summary>
+        public unsafe LinkedList ExtendedData
+        {
+            get
+            {
+                var status = NativeInterface.CiphertextElectionContext.GetExtendedData(
+                    Handle, out NativeLinkedList value);
+                if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
+                {
+                    Console.WriteLine($"GetExtendedData Error Status: {status}");
```

Comment:
Should we add a user story to implement logging?

Reply:
#136